GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/HarbingerHelper.cs

```diff
@@ -18,6 +18,7 @@ internal static class HarbingerHelper
     [
         new BuffGainCastFinder(EnterHarbingerShroud, HarbingerShroud).UsingBeforeWeaponSwap(true),
         new BuffLossCastFinder(ExitHarbingerShroud, HarbingerShroud).UsingBeforeWeaponSwap(true),
+        new BuffGainCastFinder(ImplacableFoe, ImplacableFoe).UsingOrigin(EIData.InstantCastFinder.InstantCastOrigin.Trait),
```

Comment:
That's going to proc unconditionaly on each harbinger shroud enter, I don't feel like it warrants an instant cast detection.